src/config.ts

```diff
@@ -382,3 +382,35 @@ export const TEST_LOGIN_PASSWORD = NonEmptyString.decode(
 
 // Feature flags
 export const FF_BONUS_ENABLED = process.env.FF_BONUS_ENABLED === "1";
+
+// Support Token
+export const JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY = NonEmptyString.decode(
+  process.env.JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY
+).getOrElseL(errs => {
+  log.error(
+    `Missing or invalid JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY environment variable: ${readableReport(
+      errs
+    )}`
+  );
+  return process.exit(1);
+});
+export const JWT_SUPPORT_TOKEN_ISSUER = NonEmptyString.decode(
+  process.env.JWT_SUPPORT_TOKEN_ISSUER
+).getOrElseL(errs => {
+  log.error(
+    `Missing or invalid JWT_SUPPORT_TOKEN_ISSUER environment variable: ${readableReport(
+      errs
+    )}`
+  );
+  return process.exit(1);
+});
+
+const DEFAULT_JWT_SUPPORT_TOKEN_EXPIRATION = 604800 as Second;
+export const JWT_SUPPORT_TOKEN_EXPIRATION: Second = process.env
```

Comment:
can you try to reafactor using `IntegerFromString.decode(process.env.JWT_SUPPORT_TOKEN_EXPIRATIO)` ?

Reply:
Yep